### PR TITLE
feat(chat): add possibility to upload/create(in root)/delete files in code editor (Issue #2478)

### PR DESCRIPTION
**Pull request by Alexander-Kezik**
**Description:**

add possibility to upload/create(in root)/delete files in code editor

Issues:

- Issue #2478

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2501.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2501.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2501.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2501.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2501.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2501.nightly-test.deltixhub.io
>E2E tests status: failed
>E2E tests status: success

File: apps/chat/.env.development [hunk 1]
```diff
@@ -125,3 +125,4 @@ NEXT_PUBLIC_OVERLAY_HOST="localhost:3000"
 
 ## Marketplace
 TOPICS="Business,Development,User Experience,Analysis,SQL,SDLC,Talk-To-Your-Data,RAG,Text Generation,Image Generation,Image Recognition"
+CODE_EDITOR_PYTHON_VERSIONS="python3.11,python3.12"
```
Comment (IlyaBondar) [inline apps/chat/.env.development:128]: Why not `python3.9,python3.10,python3.11,python3.12`?

File: apps/chat/src/utils/server/get-common-page-props.ts [hunk 2]
```diff
@@ -82,6 +82,8 @@ export const getCommonPageProps: GetServerSideProps = async ({
     defaultAssistantSubmodelId:
       process.env.NEXT_PUBLIC_DEFAULT_ASSISTANT_SUB_MODEL ??
       FALLBACK_ASSISTANT_SUBMODEL_ID,
+    codeEditorPythonVersions:
+      process.env.CODE_EDITOR_PYTHON_VERSIONS?.split(',') || [],
```
Comment (IlyaBondar) [inline apps/chat/src/utils/server/get-common-page-props.ts:86]: I suppose we need to provide default values instead of empty array

**Review by IlyaBondar — APPROVED** [review]

**Pull request merged**